samples/subsys/ipc/rpmsg_multi_instance/prj.conf

```diff
@@ -0,0 +1,32 @@
+CONFIG_PRINTK=y
+CONFIG_IPM=y
+CONFIG_TIMESLICE_SIZE=1
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_HEAP_MEM_POOL_SIZE=4096
+
+# Enable RPMsg multiple instance
+CONFIG_RPMSG_MULTI_INSTANCE=y
+CONFIG_RPMSG_MULTI_INSTANCE_MASTER=y
+CONFIG_RPMSG_MULTI_INSTANCES_NO=2
+
+CONFIG_IPM=y
+CONFIG_IPM_NRFX=y
```

Comment:
Similarly to ipc_service sample, this one also includes multiple entries specific to given SoC. Unless this sample is intended to be used only with this SoC, these entries should be moved to SoC-specific configuration extensions.